SETUP.md

```diff
@@ -46,3 +46,9 @@ Ensure that you have docker and docker-compose installed on your machine, as wel
               6. Restart VS Code
        6. You can also add/edit/remove rules from *eslint.json* file     
        7. Click [here](https://eslint.org/) for more details about **ESLint** or [here](https://github.com/airbnb/javascript/tree/master/react#basic-rules) to know more about the Airbnb React style
+ * Snyk
+   * Run 'npm install -g snyk' in a terminal
+   * Add snyk to the path environment variable (usually "C:\Users\<name>\AppData\Roaming\npm")
```

Comment:
User Variables or System Variables?

Reply:
I stuck it in both :P

Reply:
It could be great to specify :)

Reply:
I will work on the wording here, since its confusing

Reply:
updated in previous commit